src/animation.jl

```diff
@@ -123,7 +123,7 @@ function buildanimation(
                 `-v $verbose_level -framerate $framerate -i $(animdir)/%06d.png -i "$(animdir)/palette.bmp" -lavfi "paletteuse=dither=sierra2_4a" -loop $loop -y $fn`,
             )
         end
-    elseif file_extension(fn) == "apng"
+    elseif file_extension(fn) in ["png", "apng"]
```

Comment:
```suggestion
    elseif file_extension(fn) in ("png", "apng")
```

`Tuple`s are stack allocated, `Vector`s are heap allocated (hence less performant).